docs/dev-process/coding-guidelines.md

```diff
@@ -108,6 +108,9 @@ Some general guidelines:
 * Be aware of APIs such as `String.Split(params char[])` that do not provide overloads to avoid array allocation.
   When calling such APIs, reuse a static array when possible (e.g. `Utils.Separators.Colon`).
 
+* Avoid using string interpolations and overloads with implicit parameters like `Culture` and `StringComparison`.
+  Instead, use overloads with explicit parameters like `String.Format(IFormatProvider, String, Object[])` and `Equals(String, String, StringComparison)`.
```

Comment:
```suggestion
  Instead, use overloads with explicit parameters such as `String.Format(IFormatProvider, String, Object[])` and `Equals(String, String, StringComparison)`.
```
`like` does not sound very professional

Reply:
I am not native English. I used the pattern from the file. Could you please review the file and fix issues?

Reply:
Sure